docs/rackhd/npm_based_installation.rst

```diff
@@ -0,0 +1,242 @@
+      # Use this option to signal to the PXE client that we are doing proxy DHCP
+      option vendor-class-identifier "PXEClient";
+    }
+
+
+#######
+
+* **RACKHD APPLICATIONS**
+
+   Create the required file /opt/monorail/config.json , you can use the demonstration configuration file at https://github.com/RackHD/RackHD/blob/master/packer/ansible/roles/monorail/files/config.json as a reference.
+
+#######
+
+* **RACKHD BINARY SUPPORT FILES**
+
+   Downloaded binary files from bintray.com/rackhd/binary and placed them.
+
+   .. code::
+
+    #!/bin/bash
+
+    mkdir -p node_modules/on-tftp/static/tftp
+    cd node_modules/on-tftp/static/tftp
+
+    for file in $(echo "\
+    monorail.ipxe \
+    monorail-undionly.kpxe \
+    monorail-efi64-snponly.efi \
+    monorail-efi32-snponly.efi");do
+    wget "https://dl.bintray.com/rackhd/binary/ipxe/$file"
+    done
+
+    cd -
+
+    mkdir -p node_modules/on-http/static/http/common
+    cd node_modules/on-http/static/http/common
+
+    for file in $(echo "\
+    base.trusty.3.16.0-25-generic.squashfs.img \
+    discovery.overlay.cpio.gz \
+    initrd.img-3.16.0-25-generic \
+    vmlinuz-3.16.0-25-generic");do
+    wget "https://dl.bintray.com/rackhd/binary/builds/$file"
+    done
+
+    cd -
+
+3. **Start RackHD**
+
+   Start the 5 components of RackHD with pm2.
+  
+    .. code::
+
+       pm2 start rackhd-pm2.yml
+
+   An example of the yml:
+
+    .. code::
+
+     apps:
+        - script: index.js
+          name: on-taskgraph
+          cwd: node_modules/on-taskgraph
+        - script: index.js
+          name: on-http
+          cwd: node_modules/on-http
+        - script: index.js
+          name: on-dhcp-proxy
+          cwd: node_modules/on-dhcp-proxy
+        - script: index.js
+          name: on-syslog
+          cwd: node_modules/on-syslog
+        - script: index.js
+          name: on-tftp
+          cwd: node_modules/on-tftp
+     
+
+   All the services are started:
+
+    .. code::
+
+     ┌───────────────┬────┬──────┬───────┬────────┬─────────┬────────┬──────┬───────────┬──────────┐
+     │ App name      │ id │ mode │ pid   │ status │ restart │ uptime │ cpu  │ mem       │ watching │
+     ├───────────────┼────┼──────┼───────┼────────┼─────────┼────────┼──────┼───────────┼──────────┤
+     │ on-dhcp-proxy │ 2  │ fork │ 16189 │ online │ 0       │ 0s     │ 60%  │ 21.2 MB   │ disabled │
+     │ on-http       │ 1  │ fork │ 16183 │ online │ 0       │ 0s     │ 100% │ 21.3 MB   │ disabled │
+     │ on-syslog     │ 3  │ fork │ 16195 │ online │ 0       │ 0s     │ 60%  │ 20.5 MB   │ disabled │
+     │ on-taskgraph  │ 0  │ fork │ 16177 │ online │ 0       │ 0s     │ 6%   │ 21.3 MB   │ disabled │
+     │ on-tftp       │ 4  │ fork │ 16201 │ online │ 0       │ 0s     │ 66%  │ 19.5 MB   │ disabled │
+     └───────────────┴────┴──────┴───────┴────────┴─────────┴────────┴──────┴───────────┴──────────┘
+
+
+#######
+
+How to Erase the Database to Restart Everything
```

Comment:
Why?
Users may want to restart rackhd with a clean database after installation of rackhd based on npm .